src/qml/imports/Theme/QfCollapsibleMessage.qml

```diff
@@ -0,0 +1,89 @@
+import QtQuick 2.12
+import QtQuick.Controls 2.12
+
+import QtQuick.Controls.Material 2.12
+import QtQuick.Controls.Material.impl 2.12
+import QtGraphicalEffects 1.12
+
+import Theme 1.0
+
+Item {
+    property bool collapsed: true
+    property alias color: titleText.color
+    property alias font: titleText.font
+    property alias titleText: titleText.text
+    property alias detailsText: detailsText.text
+
+    clip: true
+    height: collapsed ? titleText.height + 1 : titleText.height + detailsText.height + 1
+    implicitHeight: height
+
+    Behavior on height {
+        NumberAnimation { duration: 100; easing.type: Easing.InQuad; }
+    }
+
+    Rectangle {
+        anchors.fill: parent
+        color: "transparent"
+        border.color: titleText.color
```

Comment:
I would use less scary color for the border tbh. Like this one, but much more white in it.

Reply:
@suricactus , done:
![image](https://user-images.githubusercontent.com/1728657/135699313-32f4e570-1c58-47d0-a053-b637463ed559.png)

Reply:
Much better. Amazing!